airbyte-integrations/connectors/source-sonar-cloud/source_sonar_cloud/spec.yaml

```diff
@@ -3,12 +3,22 @@ connectionSpecification:
   $schema: http://json-schema.org/draft-07/schema#
   title: Sonar Cloud Spec
   type: object
-  required:
-    - user_token
-    - organization
-    - component_keys
+  oneOf:
+    - required:
+      - user_token
+      - organization
+      - component_keys
+    - required:
```

Comment:
Hmm, this looks weird here — why `oneOf` but of two identical things? 

Did you mean to have two exclusive options, one for cloud, and one for self-hosted set of config vars?

Reply:
They aren't, the self-hosted version uses the 
qualifiers parameter, while the cloud-based version uses the organization parameter.